### PR TITLE
"Disable WDAC Policies": Cleanup formatting

### PR DESCRIPTION
**Pull request by sewillia-msft**
This PR performs a few list/callout-related changes to the article ["Disable Windows Defender Application Control Policies"](https://docs.microsoft.com/en-us/windows/security/threat-protection/windows-defender-application-control/disable-windows-defender-application-control-policies):  
* Move list of WDAC policy locations into "Note" callout referencing them
* Replace boldface "Note"  with DFM `[!NOTE]` tags

### CODE REVIEW

File: windows/security/threat-protection/windows-defender-application-control/disable-windows-defender-application-control-policies.md [hunk 1]
```diff
@@ -32,7 +32,6 @@ This topic covers how to disable unsigned or signed WDAC policies.
 There may come a time when an administrator wants to disable a WDAC policy. For unsigned WDAC policies, this process is simple. The method used to deploy the policy (such as Group Policy) must first be disabled, then simply delete the SIPolicy.p7b policy file from the following locations, and the WDAC policy will be disabled on the next computer restart:
 
 -   &lt;EFI System Partition&gt;\\Microsoft\\Boot\\
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-application-control/disable-windows-defender-application-control-policies.md:34]:
```suggestion
> 
> * &lt;EFI System Partition&gt;\\Microsoft\\Boot\\
> * &lt;OS Volume&gt;\\Windows\\System32\\CodeIntegrity\\
```

File: windows/security/threat-protection/windows-defender-application-control/disable-windows-defender-application-control-policies.md [hunk 2]
```diff
@@ -32,7 +32,6 @@ This topic covers how to disable unsigned or signed WDAC policies.
...
-
 -   &lt;OS Volume&gt;\\Windows\\System32\\CodeIntegrity\\
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-application-control/disable-windows-defender-application-control-policies.md:35]:
```suggestion
```

File: windows/security/threat-protection/windows-defender-application-control/disable-windows-defender-application-control-policies.md [hunk 3]
```diff
@@ -86,5 +88,4 @@ If the signed WDAC policy has been deployed using by using Group Policy, you mus
 There may be a time when signed WDAC policies cause a boot failure. Because WDAC policies enforce kernel mode drivers, it is important that they be thoroughly tested on each software and hardware configuration before being enforced and signed. Signed WDAC policies are validated in the pre-boot sequence by using Secure Boot. When you disable the Secure Boot feature in the BIOS, and then delete the file from the following locations on the operating system disk, it allows the system to boot into Windows:
 
 -   &lt;EFI System Partition&gt;\\Microsoft\\Boot\\
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-application-control/disable-windows-defender-application-control-policies.md:90]:
```suggestion
> 
> * &lt;EFI System Partition&gt;\\Microsoft\\Boot\\
> * &lt;OS Volume&gt;\\Windows\\System32\\CodeIntegrity\\
```

File: windows/security/threat-protection/windows-defender-application-control/disable-windows-defender-application-control-policies.md [hunk 4]
```diff
@@ -86,5 +88,4 @@ If the signed WDAC policy has been deployed using by using Group Policy, you mus
...
-
 -   &lt;OS Volume&gt;\\Windows\\System32\\CodeIntegrity\\
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-application-control/disable-windows-defender-application-control-policies.md:91]:
```suggestion
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review. @sewillia-msft, please commit my suggested changes. Many thanks.

**Comment by sewillia-msft** [comment]
@JohanFreelancer9, is there any specific requirement in the style guide that lists must be enclosed in quote-blocks like that? The only reason I did that for the list around L45 is because the previous statement contextualizing/referring to said list was already in a `[!NOTE]` quote-block:
image

**Comment by illfated** [comment]
Good question. I was wondering the same, because I have not seen this formatting style yet, unless it is to include it as a part of a [!NOTE] blob.

**Comment by JohanFreelancer9** [comment]
@sewillia-msft, you are absolutely correct. Sorry about that, it is totally my mistake, and thank you for pointing it out. There certainly is no such requirement and I have deleted my suggested changes. Thanks again.

**Review by JohanFreelancer9 — APPROVED** [review]

File: windows/security/threat-protection/windows-defender-application-control/disable-windows-defender-application-control-policies.md (hunk 4 above)
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-application-control/disable-windows-defender-application-control-policies.md:91]:
```suggestion
-   &lt;OS Volume&gt;\\Windows\\System32\\CodeIntegrity\\
```

**Review by illfated — APPROVED** [review]
Looks good to me. I might get back to this document at a later time for some whitespace cleanup.

**Pull request merged**